### PR TITLE
[Editorial] Find Number Of Pair of Friends

### PR DESCRIPTION
**Pull request by brebenelmihnea**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: solutions/platinum/spoj-iitkwpch.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+id: spoj-iitkwpch
+source: SPOJ
+title: IITKWPCH - Find Number Of Pair of Friends
+author: Mihnea Brebenel
+---
+
+## Explanation
+
+We can encode a number's digits using bitmasking. Maintain a frequency array of the bitmasks;
```
[the remark this answers is not in the document]
Comment (dongliu0426) [reply solutions/platinum/spoj-iitkwpch.mdx:10]: Example of a PIE solution that I think works: compute $\sum (-1)^{\text{popcount}(S)} \cdot {{f(S)}\choose{2}}$ where $f(S)$ is the number of values in the array that is a superset of $S$ (which can be computed with SOS).

[the remark this answers is not in the document]
Comment (brebenelmihnea) [reply solutions/platinum/spoj-iitkwpch.mdx:10]: technically no

[the remark this answers is not in the document]
Comment (brebenelmihnea) [reply solutions/platinum/spoj-iitkwpch.mdx:10]: But I think it's a nice introduction to supersets and subsets

[the remark this answers is not in the document]
Comment (dongliu0426) [reply solutions/platinum/spoj-iitkwpch.mdx:10]: I'm not sure if I understand the supersets and subsets part of your solution..

**Pull request closed without merging**